doc/Protocols/pr-mserv_x-road_protocol_for_management_services.md

```diff
@@ -401,6 +405,44 @@ The response echoes back the client and the server fields of the request and add
 
 An example of the client enabling request and response is given in [Annex A.8](#a8-clientenable).
 
+### 2.9 *clientRename* - Rename Security Server Client Subsystem
+
+The *clientRename* service is used to rename subsystem registered as Security Server client.
+
+The body of the client rename message (request or response) contains following fields:
+
+* **client** – identifier of the subsystem to be renamed;
+* **subsystemName** – new name for the subsystem;
+* **server** – identifier of the Security Server where the client is registered;
+* **requestId** – for responses only, unique identifier of the request that is stored in the Central Server database \[[DM-CS](#Ref_DM-CS)\].
+
+The XML Schema fragment of the client rename request body shown below.
+
+```xml
+<xsd:complexType name="ClientRequestType">
+    <xsd:sequence>
+        <xsd:element name="server" type="id:XRoadSecurityServerIdentifierType"/>
+        <xsd:element name="client" type="id:XRoadClientIdentifierType"/>
+        <xsd:element name="subsystemName" type="string" />
+        <element name="requestId" type="tns:RequestIdType" minOccurs="0"/>
+    </xsd:sequence>
+</xsd:complexType>
+```
+
+The request is sent using HTTP POST method. The content type of the request MUST be *multipart/related* and the request must contain the following MIME parts.
+
+1. X-Road SOAP request message. The message MUST contain the regular X-Road headers and the two data fields (*server*, *client*). The content type of this part MUST be *text/xml*.
+
+2. Signature of owner member of the Security Server. The MIME part must contain signature of the SOAP request message, created with the private key corresponding to a **signing certificate** of the owner member. The content type of this part must be *application/octet-stream*. Additionally, the part MUST include header field *signature-algorithm-ID* that identifies the signature algorithm. Currently supported signature algorithms are *SHA256withRSA*, *SHA384withRSA*, *SHA512withRSA*, *SHA256withRSAandMGF1*, *SHA384withRSAandMGF1*, and *SHA512withRSAandMGF1*.
```

Comment:
I think that the message should be signed by the the member that owns the subsystem to be renamed and not by the Security Server owner member just like the client registration request. Please see the `clientReg` request for details.

Also, I noticed that `clientDisable` and `clientEnable` are signed by the Security Server owner member even if they should probably be signed by the member owning the client to be enabled / disabled. However, changing the signing member now would create backwards compatibility issues so it's better to leave them as they are.

Reply:
actually it works as you said, updated docs but noticed that owner change request uses new owner for signing not the current ss owners certificate

Reply:
So it was just a documentation issue. The owner change request uses the new owner's signing certificate on purpose. The change owner management request itself is sent and signed by the current owner, and therefore, the signature and sign certificate of the new owner are included in the request attachments.